### PR TITLE
Add command Reopen Closed Tab

### PR DESCRIPTION
**Pull request by cinqmilleans**
**Resolved / Related Issues**
Add Ctrl+Shift+T hotkey to reopen last closed tab #4796
Works in horizontal and vertical mode.

**Details of Changes**
Add option "Reopen closed tab" in the tab menu
Add missing shortcut in the tab menu
The state is dynamic.

**Validation**
How did you test these changes?
- [ ] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
tabmenuh
tabmenuv

### CODE REVIEW

File: Files/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -1411,7 +1411,7 @@
     <value>Move tab to new window</value>
   </data>
   <data name="HorizontalMultitaskingControlNewTab.Text" xml:space="preserve">
-    <value>New tab (Ctrl+T)</value>
```
Comment (yaichenbaum) [inline Files/Strings/en-US/Resources.resw:1414]: Did you make this change by mistake?

File: Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml [hunk 2]
```diff
@@ -23,10 +23,13 @@
...
                             x:Uid="HorizontalMultitaskingControlNewTab"
                             Click="{x:Bind vm:MainPageViewModel.AddNewTabAtIndex}"
-                            Text="New tab (Ctrl+T)">
+                            Text="New tab">
                             <MenuFlyoutItem.Icon>
                                 <FontIcon FontFamily="{StaticResource CustomGlyph}" Glyph="&#xF113;" />
                             </MenuFlyoutItem.Icon>
+                            <MenuFlyoutItem.KeyboardAccelerators>
+                                <KeyboardAccelerator Key="T" Modifiers="Control" />
+                            </MenuFlyoutItem.KeyboardAccelerators>
```
Comment (yaichenbaum) [inline Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml:32]:
```suggestion
                            <MenuFlyoutItem.KeyboardAccelerators>
                                <KeyboardAccelerator Key="T" Modifiers="Control" IsEnabled="False" />
                            </MenuFlyoutItem.KeyboardAccelerators>
```

File: Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml [hunk 3]
```diff
@@ -51,6 +54,15 @@
                             Click="{x:Bind vm:MainPageViewModel.CloseTabsToTheRight}"
                             DataContextChanged="MenuItemCloseTabsToTheRight_DataContextChanged"
                             Text="Close tabs to the right" />
+                        <MenuFlyoutItem
+                            x:Name="MenuItemReopenClosedTab"
+                            x:Uid="HorizontalMultitaskingControlReopenClosedTab"
+                            Command="{x:Bind vm:MainPageViewModel.ReopenClosedTabCommand}"
+                            Text="Reopen closed tab">
+                            <MenuFlyoutItem.KeyboardAccelerators>
+                                <KeyboardAccelerator Key="T" Modifiers="Control,Shift" />
```
Comment (yaichenbaum) [inline Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml:63]:
```suggestion
                                <KeyboardAccelerator Key="T" Modifiers="Control,Shift" IsEnabled="False" />
```

**Review by yair100 — CHANGES_REQUESTED** [review]

**Comment by yaichenbaum** [comment]
@cinqmilleans I am not sure we want to add this option to the vertical tab flyout.

**Comment by cinqmilleans** [comment]
@yair100 If it's only horizontal, the code can be much simpler.

**Comment by yair100** [comment]
> @yair100 If it's only horizontal, the code can be much simpler.

Yeah, the vertical tab pane is a simplified version, so I don't think we need the button there.

**Comment by cinqmilleans** [comment]
@yair100 Ok, I close this pr. I'll make a new one later.

**Comment by yair100** [comment]
> @yair100 Ok, I close this pr. I'll make a new one later.

Instead of closing PRs, can you work off of the existing one?

**Pull request closed without merging**